babycenter/templates/core/section_page.html

```diff
@@ -33,13 +33,34 @@ <h3 class="article-list__title article-list__title{{self.get_effective_extra_sty
   {% if articles %}
   <div class="step-links l-indent l-indent--tall">
     {% if articles_paginated.has_previous %}
-        <a href="?p={{ articles_paginated.previous_page_number }}">&larr;</a>
+      <a href="?p={{ articles_paginated.previous_page_number }}">< Prev</a>
     {% endif %}
-    <span class="current">
-        Page {{ articles_paginated.number }} of {{ articles_paginated.paginator.num_pages }}
-    </span>
+
+      {% get_pages articles_paginated as pages %}
+        {% for page in pages.start %}
+          <a href="?p={{ page }}">{{ page }}</a>
+        {% endfor %}
+        {% if pages.ellipses_before %}
+          ...
+        {% endif %}
+        {% for page in pages.before_current %}
+          <a href="?p={{ page }}">{{ page}}</a>
+        {% endfor %}
+        <span class="current">
+            {{ articles_paginated.number }}
+        </span>
+        {% for page in pages.after_current %}
+          <a href="?p={{ page }}">{{ page }}</a>
+        {% endfor %}
+        {% if pages.ellipses_after %}
+          ...
+        {% endif %}
+        {% for page in pages.end %}
+          <a href="?p={{ page }}">{{ page }}</a>
+        {% endfor %}
```

Comment:
Could we de-indent this pagination stuff to be consistent with the rest of the indentation?